pkg/lightning/restore/restore.go

```diff
@@ -2221,8 +2209,8 @@ func (tr *TableRestore) analyzeTable(ctx context.Context, g glue.SQLExecutor) er
 ////////////////////////////////////////////////////////////////
 
 var (
-	maxKVQueueSize         = 128   // Cache at most this number of rows before blocking the encode loop
-	minDeliverBytes uint64 = 65536 // 64 KB. batch at least this amount of bytes to reduce number of messages
+	maxKVQueueSize         = 128            // Cache at most this number of rows before blocking the encode loop
+	minDeliverBytes uint64 = 96 * units.KiB // 96 KB (data + index). batch at least this amount of bytes to reduce number of messages
```

Comment:
Is this change has side effect for tidb backend because we increase the batch size?🤔

Reply:
the TiDB backend does `SplitIntoChunks` inside `WriteRows`, so fortunately no.